micro_epsilon_scancontrol_driver/include/micro_epsilon_scancontrol_driver/driver.h

```diff
@@ -37,7 +38,7 @@ namespace scancontrol_driver
     {
         public:
             // Constructor and destructor
-            ScanControlDriver();
+            ScanControlDriver(const std::string& name);
             ~ScanControlDriver() {}
```

Comment:
issue: This should probably be marked as `default` instead of creating an empty function.